src/img/node_modules/imgmanifest/package.json

```diff
@@ -3,8 +3,8 @@
   "description": "Library for working with Triton/SDC/SmartOS image manifests",
   "version": "3.1.0",
```

Comment:
This has already been updated in the source repo: https://github.com/TritonDataCenter/node-imgmanifest

I'm not sure if we have a special process for updating the modules in this repo since we commit the node_modules directory in this repo.

The correct answer for this package.json will be true for every package.json within node_modules, I suspect most, if not all, have been updated in their source repo.